js/src/dapps/registry/Names/actions.js

```diff
@@ -15,22 +15,28 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 import { sha3, api } from '../parity.js';
+import { getOwner, isOwned } from '../util/registry';
 import postTx from '../util/post-tx';
 
+export const clearError = () => ({
```

Comment:
If `clearError` is getting handled app-wide (in `dapps/registry/reducers`), why not declare it in `dapps/registry/actions`?